source/features/git-checkout-pr.tsx

```diff
@@ -51,9 +53,9 @@ function checkoutOption(remote?: string, remoteType?: 'HTTPS' | 'SSH'): JSX.Elem
 					className="copyable-terminal-content"
 				>
 					<span className="user-select-contain">
-						{remote && `git remote add ${remote} ${connectionType[remoteType!]}${getPRHeadRepo()!.nameWithOwner}.git\n`}
-						git fetch {remote ?? 'origin'} {getCurrentBranch()}{'\n'}
```

Comment:
Usage of `getCurrentBranch` in PRs was ambiguous (there are 2 branches) and not it completely throws when used. It's best to use the splitter above. It can be made into its own helper later.